src/arithmetic.jl

```diff
@@ -96,6 +96,33 @@ end
     end
 end
 
+# Mixtures
+@inline isless(a::Taylor1{TaylorN{T}}, b::Taylor1{TaylorN{T}}) where {T<:NumberNotSeries} =
+    isless(a - b, zero(T))
+@inline function isless(a::HomogeneousPolynomial{Taylor1{T}}, b::HomogeneousPolynomial{Taylor1{T}}) where {T<:NumberNotSeries}
```

Comment:
What is the reason for including the `@inline` macro?

Reply:
Also the `NumberNotSeries` seems very anti-Julian in that it prohibits the use of different number types (e.g. `Dual` or `Unitful`). I would rather see deriving from `<:Number` and explicity writing the unimplemented method for `{T<:Series}` to raise an error.

Reply:
This is indeed a fair criticism, yet beyond this PR. I opened #325 to remember it...